mysql-dao/src/main/java/com/kovan/service/DocumentService.java

```diff
@@ -0,0 +1,45 @@
+package com.kovan.service;
+
+import com.kovan.entity.Document;
+import com.kovan.repository.DocumentRepository;
+import org.springframework.stereotype.Service;
+import static java.time.Instant.now;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class DocumentService {
+
+    private final DocumentRepository documentRepository;
+
+    public DocumentService(DocumentRepository documentRepository) {
+        this.documentRepository = documentRepository;
+    }
+
+    public String saveDocument(Document document) {
+
+        documentRepository.save(Document.builder()
```

Comment:
why do we need this builder here already you are passing entity object so you could do whatever operation you want in service itself

Reply:
removed builder method.